assets/sass/_typography.scss

```diff
@@ -57,9 +57,15 @@ Style guide: Typography.1 Typeface.1 Font stacks
 /*
 Accessibility
 
-- Don't write in all *italics* or CAPITALS.
-- Avoid using **bold**. Bolding a few words may help scannability, **but blocks of bold are harder to read**.
 - Use just 1 font (Open Sans).
+- Write short sentences. Make line length less than 75 characters. Use grid boxes to limit length.
+- Set `line-height` (leading) to at least 1.5.
```

Comment:
This shouldn’t be necessary, since our setup provides it; other devs extending UI-Kit would only be overwriting it, which they shouldn’t.